src/utils/decodeB64ToUint8Array.js

```diff
@@ -1,7 +1,7 @@
-import window from 'global/window';
+import atob from 'atob';
```

Comment:
this becomes `import atob from 'window.atob'` in the browser with rollup globals.